cluster/centos/master/scripts/apiserver.sh

```diff
@@ -43,8 +43,8 @@ KUBE_ETCD_KEYFILE="--etcd-keyfile=/srv/kubernetes/etcd/client-key.pem"
 # --insecure-bind-address=127.0.0.1: The IP address on which to serve the --insecure-port.
 KUBE_API_ADDRESS="--insecure-bind-address=0.0.0.0"
 
-# --insecure-port=8080: The port on which to serve unsecured, unauthenticated access.
-KUBE_API_PORT="--insecure-port=8080"
+# --secure-port=8080: The port on which to serve unsecured, unauthenticated access.
```

Comment:
2 nitpicks here:

1) `8080` port is used for http see: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers
2)  The sentence `... to serve unsecured, unauth..`  need to be modified for `secure` mode

Reply:
+1, We can use the default port `6443`
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/